pages/participantinfo.tsx

```diff
@@ -0,0 +1,516 @@
+                <Stack direction="column">
+                    <Checkbox>Physiotherapy</Checkbox>
+                    <Checkbox>Speech and Language Therapy</Checkbox>
+                    <Checkbox>Occupational Therapy</Checkbox>
+                    <Checkbox>Psychotherapy/Counseling</Checkbox>
+                    <Checkbox>Music or Art Therapy</Checkbox>
+                    <Checkbox>Other</Checkbox>
+                </Stack>
+            </FormControl>
+            <FormControl id="parent/guardian-expectations">
+                <FormLabel>Parent/Guardian Expectations</FormLabel>
+                <Textarea placeholder="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Facilisi mauris enim, egestas." />
+            </FormControl>
+        </FormPage>,
+        <FormPage>
+            <FormLabel>
+                {" "}
+                Parent/Guardian Name
+                <HStack spacing="24px">
+                    <FormControl id="first-name">
+                        <Input placeholder="First name" />
+                    </FormControl>
+                    <FormControl id="last-name">
+                        <Input placeholder="Last name" />
+                    </FormControl>
+                </HStack>
+            </FormLabel>
+            <FormControl id="phone-number">
+                <FormLabel>Phone Number </FormLabel>
+                <Input placeholder="289 349 1048" />
+            </FormControl>
+            <FormControl id="relationship-to-participant">
+                <FormLabel>Relationship to Participant</FormLabel>
+                <Input placeholder="Mother" />
+            </FormControl>
+        </FormPage>,
+        <FormPage>
+            <Box maxW="55rem">
+                <Text noOfLines={3} fontSize="16px" fontWeight="200">
+                    The information on this form will be used at the discretion
+                    of the activity instructor/coordinator to ensure care and
+                    attention is given to the safety and health of your child.
+                    All information on this form is considered Personal and
+                    Confidential. The contact listed on the emergency form
+                    cannot be the same contact listed as the parent above.
+                </Text>
+            </Box>
+            <FormLabel>
+                {" "}
+                Emergency Contact Name
+                <HStack spacing="24px">
+                    <FormControl id="first-name">
+                        <Input placeholder="First name" />
+                    </FormControl>
+                    <FormControl id="last-name">
+                        <Input placeholder="Last name" />
+                    </FormControl>
+                </HStack>
+            </FormLabel>
+            <FormControl id="emergency-contact-cell-number">
+                <FormLabel>Emergency Contact Cell Number </FormLabel>
+                <Input placeholder="289 349 1048" />
```

Comment:
For some reason when I enter a value for the "relationship to participant" field in the previous page, that value shows up in the emergency contact cell number. I'll try to look into why that's the case

Reply:
Fixed: https://github.com/uwblueprint/social-diversity-for-children/pull/43/commits/9c818dda1922d59cfb184bd23c297e88c0f7b4cb